gestalt/vault.py

```diff
@@ -275,14 +228,16 @@ def _validate_token_expiration(self) -> None:
             else:
                 expire_time = expire_time.astimezone(timezone.utc)
 
-            threshold = timedelta(days=EXPIRATION_THRESHOLD_DAYS)
             current_time = datetime.now(timezone.utc)
-            delta_time = expire_time - current_time
+            # in hours
+            delta_time = (expire_time - current_time).total_seconds() / 3600
 
-            if delta_time <= threshold:
-                print(f"Re-auth with vault.")
+            if delta_time < EXPIRATION_THRESHOLD_HOURS:
```

Comment:
why did we change expiration threshold to just 1 hour?

Reply:
While testing I found that most of the services have a 3h ttl. I want to reduce the amount of calls we make for re-authenticate for token that have less than 1h expiration time.